test/test_helper.rb

```diff
@@ -160,3 +160,11 @@ def restore_plugins
   @klass.clear
   @klass.instance_variable_set('@registered_plugins', @plugins_backup)
 end
+
+def with_auditing(klass)
+  auditing_was_enabled = klass.auditing_enabled
+  klass.enable_auditing
+  yield
+ensure
+  klass.enable_auditing if auditing_was_enabled
```

Comment:
I think @tbrisker has a valid point here, do we want to revert this to the original state?

Reply:
Again, this method was taken from audited gem, there is no `enable_auditing=` method available. Please elaborate what's wrong.

Reply:
I am wondering if `klass.disable_auditing unless auditing_was_enabled` is missing.

Reply:
I think this is still not resolved, original value is not being restored in ensure, I think this would be necessary `"auditing_was_enabled ? klass.enable_auditing : klass.disable_auditing"` in ensure block

perhaps link audited code you copied here?

Reply:
NOW I get it! This code was from audited gem, but they had the opposite `with_auditing". Looks like I only need this then: `klass.disable_auditing unless auditing_was_enabled`